src/main/java/frc/robot/subsystems/LimeVision/LimeLightSub.java

```diff
@@ -100,6 +174,27 @@ public double getAngle() {
     return angle;
   }
 
+  public double[] getBotPose() {
```

Comment:
I believe we are not using this correct? We are using the new bot pose blue instead. We should remove this to avoid confusion.